nyx_client/nyx_client/client.py

```diff
@@ -554,13 +555,14 @@ def create_data(
         size: int,
         genre: str,
         categories: Sequence[str],
-        download_url: str,
         content_type: str,
         lang: str = "en",
         status: str = "published",
         preview: str = "",
         price: int | None = None,
         license_url: str | None = None,
+        download_url: Optional[str] = None,
+        file: Optional[BufferedReader] = None,
```

Comment:
`RawIOBase`? (+ use `|` instead of `Optional`)

Reply:
After more thought I think `RawIOBase` is maybe more appropriate - we don't really want to trigger decoding of their file in the client. That will cover text & non-text.